src/components/TransactionPanel/TransactionPanel.helpers.js

```diff
@@ -106,7 +106,7 @@ export const AddressLinkMaybe = props => {
   const fullAddress =
     typeof building === 'string' && building.length > 0 ? `${building}, ${address}` : address;
 
-  return (isProvider || txIsAcceptedForCustomer) && hrefToGoogleMaps ? (
+  return txIsAcceptedForCustomer && hrefToGoogleMaps ? (
```

Comment:
Hmm. A provider could see the location of his listing all the time, but this works too.

Reply:
@Gnito I don't quite follow. The idea is to hide the address link from the provider.

Reply:
@lyyder provider owns the listing - he knows where it is.

Reply:
Yes, and that's why it's not relevant to show the address link to the provider.

Reply:
Solved IRL: When to show the listing location link has been a topic of a debate also before and probably continues to be one in the future. For now, we'll only show it only to customers after a booking has been accepted.